### ISSUE TITLE
Update dependency to latest version

### ISSUE THREAD

**Issue by tab00** [JackAdams/meteor-transactions#97]
socialize:server-time is now at version 1.0.0.

**Comment by JackAdams** [comment]
Yes, I tried updating that dependency but ran into some trouble with imports, if I remember right. Might just remove that dependency altogether at some stage.
